validate/capabilities/validate_linux.go

```diff
@@ -0,0 +1,19 @@
+//go:build linux
+// +build linux
+
```

Comment:
nit: this is redundand, since there's already `_linux` in the file name.

Reply:
Removed build-tags.